src/DevTools/createMockNetworkLayer/index.ts

```diff
@@ -1,14 +1,37 @@
-import { IMocks } from "graphql-tools/dist/Interfaces"
+import { GraphQLFieldResolver, responsePathAsArray } from "graphql"
+import { IResolvers } from "graphql-tools/dist/Interfaces"
 import getNetworkLayer from "relay-mock-network-layer"
 import { Network } from "relay-runtime"
 import schema from "../../../data/schema.graphql"
 import FormattedNumber from "./CustomScalars/formatted_number"
 
-export const createMockNetworkLayer = (mockResolvers: IMocks) => {
+const complain = (info, type) => {
+  const path = responsePathAsArray(info.path).join("/")
+  const message = `A mock for field at path '${path}' of type '${type}' was expected but not found.`
+  console.error(message)
+  throw new Error(message)
+}
+
+export const createMockNetworkLayer = (mockResolvers: IResolvers) => {
```

Comment:
I wonder if we should even allow specifiying resolvers rather than just an object of data, because the resolvers are hard to understanding for people and if we emir types for the expected data the resolvers functions wouldn’t be part of that either.

I think we should rename this new implementation and leave the previous one in place and deprecate it. The new version can then only accept pure data objects. If it all works out, then we can probably even drop the graphql-tools dependency in the future and use just the graphql-JS API

Reply:
Would it never be desirable to have a resolver return different items during the same query?

Reply:
That’s what I’m starting to question and so think this transition could be a good time to tests that out. I think pure data objects are best understood by all for fixture usage.